spec/components/standup_meeting_group/check_in_status_component_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe StandupMeetingGroup::CheckInStatusComponent, type: :component do
+  let(:user) { create(:user) }
+
+  let(:standup_meeting_group) do
+    create(:standup_meeting_group).tap do |group|
+      group.standup_meetings << create(:standup_meeting, user:, status: 'completed')
+    end
+  end
+  let(:standup_meeting_group2) { create(:standup_meeting_group) }
```

Comment:
I would maybe name these standup meeting groups differently. Maybe something like `standup_meeting_group_with_checkin` and `standup_meeting_group_no_checkin` or something. Just something that gives more information about how they're different at a glance.

This isn't really a blocker though